pt-BR/mail/chrome/messenger/preferences/advanced.dtd

```diff
@@ -59,85 +59,85 @@
 <!ENTITY showUpdates.accesskey          "H">
 
 <!-- Networking and Disk Space -->
-<!ENTITY showConnections.label         "Connection...">
+<!ENTITY showConnections.label         "Conexão...">
 <!ENTITY showConnections.accesskey     "n">
 
 <!ENTITY offlineCompact.label "Condensar pastas se puder liberar pelo menos">
 <!ENTITY offlineCompact.accesskey "n">
 <!ENTITY kb.label "KB">
 
 <!-- Certificates -->
-<!ENTITY certSelection.label "When a server requests my certificate:">
-<!ENTITY certs.auto                      "Select one automatically">
-<!ENTITY certs.ask                       "Ask me every time">
+<!ENTITY certSelection.label "Quando um servidor solicitar meu certificado:">
+<!ENTITY certs.auto                      "Selecionar um automaticamente">
+<!ENTITY certs.ask                       "Sempre perguntar">
 
 <!ENTITY viewCertificates.label         "Certificados">
 <!ENTITY viewCertificates.accesskey     "C">
 <!ENTITY viewCRLs.label                 "Revogações">
 <!ENTITY viewCRLs.accesskey             "R">
-<!ENTITY viewOCSP.label                 "Verification">
+<!ENTITY viewOCSP.label                 "Verificação">
 <!ENTITY viewOCSP.accesskey             "f">
 <!ENTITY viewSecurityDevices.label      "Dispositivos de segurança">
 <!ENTITY viewSecurityDevices.accesskey  "D">
 
 <!-- Web Services -->
-<!ENTITY webservicesInfo.label          "Configure the Web services used within Postbox.">
+<!ENTITY webservicesInfo.label          "Configurar os Web services usados dentro do Postbox.">
```

Comment:
CORRECTIONS NEEDED it should be <!ENTITY webservicesInfo.label          "Configurar os serviços web usados dentro do Postbox.">